app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,37 @@
+class TaskController < ApplicationController
```

Comment:
You have a naming problem

![Uploading image.png…]()


```suggestion
class TasksController < ApplicationController
```

Reply:
I solved some problems in a couple and I realized that I don’t know how to write :)